code/modules/keybindings/_keybindings.dm

```diff
@@ -0,0 +1,22 @@
+/datum/keybinding
+	var/list/hotkey_keys
+	var/list/classic_keys
+	var/name
+	var/full_name
+	var/description = ""
+	var/category = CATEGORY_MISC
+	var/weight = WEIGHT_LOWEST
+
+/datum/keybinding/New()
+	// Default keys to the master "hotkey_keys"
+	if(LAZYLEN(hotkey_keys) && !LAZYLEN(classic_keys))
+		classic_keys = hotkey_keys.Copy()
+
+/datum/keybinding/proc/down(client/user)
+	return FALSE
```

Comment:
What do these return values mean?

Reply:
I didn't see any related code which can work with it. Probably, it was designed to "confirm" what down button is successfully pressed.

Reply:
It needed because keypress verb uses check, and check for true return.